src/config/fix_inversion.json

```diff
@@ -582,8 +582,926 @@
+            "invert": "img[src]"
+        },
+        {
+            "url": "vuejs.org",
+            "invert": [
+                ".content, pre, h3, p"
+            ],
+            "noinvert": [
+                "img"
+            ]
+        },
+        {
+            "url": "web.whatsapp.com",
+            "invert": ".qrcode .icon-logo",
+            "rules": ".qrcode { border: 8px solid black; }"
+        },
+        {
+            "url": "wikipedia.org",
+            "invert": ".mwe-popups-discreet > svg",
+            "noinvert": [
+                ".mwe-math-fallback-image-inline",
+                ".mwe-math-fallback-image-display"
+            ]
+        },
+        {
+            "url": "wikiwand.com",
+            "removebg": "img"
+        },
+        {
+            "url": "wolfram.com",
+            "noinvert": [
+                ".numberedequation, .displayformula, .inlineformula"
+            ]
+        },
+        {
+            "url": "yandex.*",
+            "invert": [
+                ".afisha, #wd-_afisha",
+                ".favicon",
+                ".informers7__icon",
+                ".places__content-wrapper",
+                ".schedule__icon",
+                ".stream-intro",
+                ".zen__item-domain",
+                ".zen__item-title",
+                ".button_theme_white",
+                ".stream-intro__items",
+                ".services__icon",
+                ".teaser__service-bg",
+                ".places__card",
+                ".menu2__wrapper",
+                ".footer",
+                ".logo-service__image",
+                ".app-icon",
+                ".services-big__item_icon",
+                ".services-all__icon",
+                "[class*='icon_moon_']",
+                ".weather__icon, .traffic__icon",
+                ".media-grid_theme_white, .dist-popup__image"
+            ],
+            "noinvert": [
+                ".afisha__film-image",
+                ".stream-intro__vod-preview"
+            ],
+            "removebg": [
+                ".stream-intro__items",
+                ".mini-suggest__popup-spacer"
+            ],
+            "rules": [
+                ".body__wrapper > .i-bem {background-color: #fff}",
+                ".stream-intro__items, .block__title * {color: #000 !important}"
+            ]
+        },
+        {
+            "url": "youtube.com",
+            "invert": [
+                "html[dark=true] #scrim",
+                ".html5-video-player, .has-custom-banner, #theater-background",
+                "#watch-appbar-playlist"
+            ],
+            "noinvert": [
+                "html[dark=true]",
+                "html[dark=true] *",
+                ".html5-video-player *",
+                ".has-custom-banner *",
+                "#watch-appbar-playlist *"
+            ],
+            "rules": [
+                "#scrim.app-drawer { background: rgba(255, 255, 255, 0.5); }"
+            ]
+        }
+    ]
+}
```

Comment:
Here is an json format error. It's expecting EOF, instead of another object entry.